template.tpl

```diff
@@ -619,6 +763,167 @@ ___TEMPLATE_PARAMETERS___
+                      {
+                        "paramName": "detAttribution",
+                        "paramValue": true,
+                        "type": "EQUALS"
+                      }
+                    ]
+                  }
+                ],
+                "help": "Check this box to enable attribution tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-marketing-attribution\"\u003eRead more\u003c/a\u003e.",
+                "defaultValue": true
+              },
+              {
+                "type": "CHECKBOX",
+                "name": "detPageView",
+                "checkboxText": "Enable page views tracking",
+                "simpleValueType": true,
+                "subParams": [
+                  {
+                    "type": "GROUP",
+                    "name": "pageViewOptions",
+                    "groupStyle": "NO_ZIPPY",
+                    "subParams": [
+                      {
+                        "type": "CHECKBOX",
+                        "name": "pageViewLegacy",
+                        "checkboxText": "Use the legacy page view properties",
+                        "simpleValueType": true,
+                        "subParams": [],
+                        "help": "Enable this to use legacy page view event type (\u003cstrong\u003ePage View\u003c/strong\u003e) and event properties (\u003cstrong\u003epage_location\u003c/strong\u003e, \u003cstrong\u003epage_path\u003c/strong\u003e, \u003cstrong\u003epage_title\u003c/strong\u003e, \u003cstrong\u003epage_url\u003c/strong\u003e). By default it will use the `Page View Type` input as the event type and \u003cstrong\u003e[Amplitude] Page Domain\u003c/strong\u003e, \u003cstrong\u003e[Amplitude] Page Location\u003c/strong\u003e, \u003cstrong\u003e[Amplitude] Page Path\u003c/strong\u003e, \u003cstrong\u003e[Amplitude] Page Title\u003c/strong\u003e. \u003cstrong\u003e[Amplitude] Page URL\u003c/strong\u003e as the event properties for the page view event.",
+                        "defaultValue": false
+                      },
+                      {
+                        "type": "TEXT",
+                        "name": "pageViewType",
+                        "displayName": "Page View Type",
+                        "simpleValueType": true,
+                        "help": "The event type for page view event. The default value has been change from \u003cstrong\u003ePage View\u003c/strong\u003e to \u003cstrong\u003e[Amplitude]  Page Viewed\u003c/strong\u003e.",
+                        "enablingConditions": [
+                          {
+                            "paramName": "pageViewLegacy",
+                            "paramValue": false,
+                            "type": "EQUALS"
+                          }
+                        ],
+                        "defaultValue": "[Amplitude] Page Viewed"
```

Comment:
I think it's neater if the default value in the form is an empty string then we just handle the default resolution behind the scenes